### ISSUE TITLE
[Bug]  Crystalite armor

### ISSUE THREAD

**Issue by crowscythe13** [quiqueck/BetterEnd#446]
### What happened?

A bug happened! So i had my friends give me the termite ingots since i couldn't use the end smelter because i wanted the Crystalite armor, sadly for whatever reason an armor set that is made from diamond equivalent armor gives less armor. only the chest piece was giving armor to me making it so I had only 3 armor bars/ 6 armor. I like to think this is a bug and not something that is intended in the game. 

### BetterEnd

21.0.5

### BCLib

21.0.9

### Minecraft

1.20.4

**Comment by xCykrix** [comment]
Duplicate of #442 and should be fixed by #437 when it is merged.

**Comment by OpenBagTwo** [comment]
[To elaborate](https://github.com/quiqueck/BetterEnd/pull/437#issuecomment-2241797851), these are the intended stats:

> #### Crystalite
> 
> Item   | Armor | Knockback Resist | Toughness
> |---|---|---|---
> Helmet   | 3     | *   | 1.2
> Chestplate  | 8     | *   | 1.2
> Leggings    | 6     | *   | 1.2
> Boots  | 3     | *   | 1.2
> Elytra | 6     | *   | 0.96
>
> *These were 1.0 previously, and are zero now. I took this as an intentional nerf. If that was not the case, I'm happy to change them back to 1.

That's on par with Diamond for defense, with less toughness but possibly Netherite-level KB resistance.

But also bear in mind that each armor item gives an effect or effect resistance when worn (or, at least, will again once my PR is merged), which (IMO) more than offsets them being sub-Netherite (or even sub-Diamond) in terms of pure stats.
